go.mod

```diff
@@ -337,5 +341,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-// the replacement build is based on https://github.com/dragonchaser/cs3apis/tree/master
-replace github.com/cs3org/go-cs3apis => github.com/dragonchaser/go-cs3apis v0.0.0-20230918130959-ae732d4b8147
+replace github.com/cs3org/go-cs3apis => github.com/aduffeck/go-cs3apis v0.0.0-20231009082215-ad45e19edac0
```

Comment:
please re-add the comment, otherwise the next person will be confused (at least as long as we are working with our forks)